interfaces/Portal/src/app/program/program-people-affected/program-people-affected.component.html

```diff
@@ -173,6 +173,7 @@
         style="display: flex"
       >
         <button
+          data-testid="information-popup-button"
           *ngIf="canViewPersonalData"
           type="button"
```

Comment:
I'd prefer to add these properties in "order of importance", so:

```suggestion
          *ngIf="canViewPersonalData"
          type="button"
          data-testid="information-popup-button"
```

Then you first see: "should this thing even be here or not" (based on the `if`)
Then; the most-important/default attributes; (could be more then only this `type`, in this case)
Then; Other specific attributes (like this `testid` one).
Then; Other, 'less critical' properties like styles/class/et.

In this example, the other properties below could also be reused according to all kinds of discussion, sot that is not valuable to change/discuss here/now;
But the thing you add here, should not be "jumping the line" so that it 'hides' the 'important `if`-condition.. in a way.